### PR TITLE
Getting Started With Kali Linux

### LINKED ISSUE #2061: Getting started with Kali Linux
**Introduction**

Kali linux is a penetration testing platform that provides variety of security auditing tools used by hackers and security specialist in day to day encounters.

Tools available on Kali Linux enables the  users to gather information, perform exploits and prevent their devices from exploits. The article proposed will help the user to navigate and relate with major tools available on Kali Linux.

**Key Takeaways**

By the end of the article, the user will be able to:
1. Update to latest version.
2. Navigate through Kali Linux using the terminal.
3. Run major commands on Kali Linux
4. Relate with major tools.

**Reference**

N/A

### PR DESCRIPTION
**Pull request by Kimosop-Wakanda**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
This fixes #2061

### CODE REVIEW

**Comment by Bayler** [comment]
Hi @Kimosop-Wakanda Thank you for submitting this article. Please make sure your article checklist is in check before submitting to avoid unnecessary corrections. 

1. Your hero image is too big. The article checklist states that the hero image should be 600 by 400 pixels. Also rename it from hero.jpg.jpg to hero.jpg
2. Only header h3 and above (h4, h5) etc is recommended for titles. I've noticed you've used a header h2 header.

Please make those changes first before I can continue with the review.

**Comment by Kimosop-Wakanda** [comment]
Changes done @Bayler Please continue with the review

**Comment by Bayler** [comment]
> Changes done @Bayler Please continue with the review

Okay. Noted ✅

File: content/articles/getting-started-with-kali-linux/index.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
+### Getting started with Kali Linux
+
+**Kali Linux** is the most popular penetration testing platform that provides a variety of security auditing tools used by security specialists and hackers in day-to-day encounters.
+
+Tools available on Kali Linux enable the user to gather information, perform exploits and prevent their devices from exploits.
+
+### Installing Kali Linux
+
+For beginners, it is advised to have your Kali Linux in a **virtual box**.
```
Comment (Bayler) [inline content/articles/getting-started-with-kali-linux/index.md:9]: Why is it advisable to use a virtual box for beginners? You need to explain why.

File: content/articles/getting-started-with-kali-linux/index.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+### Installing Kali Linux
+
+For beginners, it is advised to have your Kali Linux in a **virtual box**. 
+You can have your Kali Linux in your favorite virtual box. Available virtual boxes include:
+
+  * Vmware virtual box
+  * Hyper V virtual box
+  * Kernel-based virtual box 
+
```
Comment (Bayler) [inline content/articles/getting-started-with-kali-linux/index.md:15]: Please add links to these virtual boxes.

File: content/articles/getting-started-with-kali-linux/index.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+  * Kernel-based virtual box 
+  
+For our tutorial, we will use VMware virtual box (My favorite).
+
+#### Download Kali Linux iso
+
+First, before installing the VMware you have to download the Kali Linux iso. Use the link below:
+
+  * Kali Linux iso.
+
```
Comment (Bayler) [inline content/articles/getting-started-with-kali-linux/index.md:23]: Where is the link? I assume the link was to take the readers to where they can download it. Please add it

File: content/articles/getting-started-with-kali-linux/index.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+#### Download Vmware virtual box
+After successfully downloading the kali Linux iso, you can now download the VMware virtual box using the link below.
+
+  * Vmware virtual box.
+
```
Comment (Bayler) [inline content/articles/getting-started-with-kali-linux/index.md:28]: This too needs to be a link

File: content/articles/getting-started-with-kali-linux/index.md [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+#### Updating Kali Linux
+
+It is recommended to frequently update your Kali Linux and its tools to the latest versions to keep it up to date. 
+Follow the steps below to update and upgrade Kali Linux:
+
+  * Start the **terminal** by:
+        
+       * Clicking the **cltr+alt+T** buttons simultaneously or
+       * Using the GUI(Graphical User Interface), go to the applications section and double click on the terminal option.
+
```
Comment (Bayler) [inline content/articles/getting-started-with-kali-linux/index.md:57]: This is misleading. Are we really clicking cltr+alt+T ? Press or hold is preferred in this case.

File: content/articles/getting-started-with-kali-linux/index.md [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+   1. **Information gathering and Vulnerability analysis**
+   
+Tools available: `Nmap`, `Zenmap` etc.
+
+   * **Nmap**
+
+Let's discuss Nmap since it is the commonly used information-gathering tool.
+
+Nmap is a tool that gathers information by scanning the target's IP address or URL. You can learn more about Nmap scanning and interpretation from the "Introduction to Nmap" tutorial.
+
```
Comment (Bayler) [inline content/articles/getting-started-with-kali-linux/index.md:215]: Please add the link to the "Introduction to Nmap" tutorial

File: content/articles/getting-started-with-kali-linux/index.md [hunk 7]
```diff
@@ -0,0 +1,290 @@
...
+   * **John the Ripper**
+
+`John the Ripper` is a free password cracking tool originally made for Unix but now available on Kali Linux.
+It works by use of dictionary method to perform brute force on the target.
+
+You can access various tools available on John the Ripper by typing in the keywords eg `john`, `maller`, `unshadow`, `unique` etc.
+
+You can learn
+
```
Comment (Bayler) [inline content/articles/getting-started-with-kali-linux/index.md:234]: This sentence seems unfinished.

File: content/articles/getting-started-with-kali-linux/index.md [hunk 8]
```diff
@@ -0,0 +1,290 @@
...
+   6. **Post Exploitation**
+
+Tools available: `Powersploit`, `Searchsploit`
+
+   * **Powerploit**
+
+This is a series of Powershell scripts that comes in handy during a post-exploitation scenario in an authorized penetration testing. They can be used in code execution, script modification, persistence, and bypass an anti-virus.
+ 
+ You can learn more about Powersploit in the article linked.
+
```
Comment (Bayler) [inline content/articles/getting-started-with-kali-linux/index.md:283]: Please add the links to all the articles mentioned

File: content/articles/getting-started-with-kali-linux/index.md [hunk 9]
```diff
@@ -0,0 +1,290 @@
...
+This is a series of Powershell scripts that comes in handy during a post-exploitation scenario in an authorized penetration testing. They can be used in code execution, script modification, persistence, and bypass an anti-virus.
+ 
+ You can learn more about Powersploit in the article linked.
+ 
+ ### Conclusion
+ 
+ In this article, we have learned:
+ - Installing Kali Linux in a Virtual Box.
+ - Updating Kali Linux to the latest version and running some major commands.
+ - Major tools used in Kali Linux.
```
Comment (Bayler) [inline content/articles/getting-started-with-kali-linux/index.md:289]: This is more of a  summary than a conclusion. Please add a conclusive statement.

**Comment by Bayler** [comment]
@Kimosop-Wakanda Thank you for submitting this article. It is a nice read. However, you have a few corrections. Please work on them then we can proceed for final review.

**Comment by Kimosop-Wakanda** [comment]
I have made all the corrections @Bayler 
Thanks for the review.

File: content/articles/getting-started-with-kali-linux/index.md (hunk 5 above)
Comment (Bayler) [reply content/articles/getting-started-with-kali-linux/index.md:57]: @Kimosop-Wakanda  You haven't edited this bit

**Comment by Kimosop-Wakanda** [comment]
Changes made @Bayler

**Comment by Bayler** [comment]
Hi @Kimosop-Wakanda this article is now ready for final review.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
@Kimosop-Wakanda great work 

Nice tutorial to follow along

Please go over any edits I made to improve future articles. 

cc @Bayler

**Pull request merged**